packages/snap-controller/src/Autocomplete/AutocompleteController.ts

```diff
@@ -338,8 +338,13 @@ export class AutocompleteController extends AbstractController {
 			},
 			keyUp: (e: KeyboardEvent): void => {
 				// ignore enter and escape keys
-				if (e?.keyCode == KEY_ENTER || e?.keyCode == KEY_ESCAPE) return;
-
+				if (e?.keyCode == KEY_ENTER || e?.keyCode == KEY_ESCAPE) {
+					if (e.keyCode == KEY_ENTER) {
+						const input = e.target as HTMLInputElement;
```

Comment:
Maybe a question for @korgon 
Do you think the term that we save to history should be what was typed in the input or the suggested term? 
Or maybe only use the user typed in term if integrated spell correction is enabled?

Reply:
Yea, we should use the term (query) that we send with our redirect to the search page - in any case when using our autocomplete the first suggested term will contain the query that we direct to the search page with.